src/spikeinterface/core/recording_tools.py

```diff
@@ -118,22 +119,27 @@ def get_closest_channels(recording, channel_ids=None, num_channels=None):
     return np.array(closest_channels_inds), np.array(dists)
 
 
-def get_noise_levels(recording, return_scaled=True, **random_chunk_kwargs):
+def get_noise_levels(recording: 'BaseRecording', return_scaled: bool = True, method: Literal['mad', 'std'] = "mad", **random_chunk_kwargs):
```

Comment:
Why BaseRecording and not the actual class?

Reply:
Why I mean here, is why quotes.
`BaseRecording` vs BaseRecording. You are getting a circular import error?

Does `BaseRecording` by itself just works?

Reply:
Yes, the actual class causes a circular import error